kcharles52/books-invetory
# chore(apollo-clinet): setup apollo client

**What does this PR do?**

- set the environment to enable making graphql queries to the graphql API

**Description of Task to be completed?**

- install apollo packages
- create apollo client

**How should this be manually tested?**

- Run the following commands
- `git fetch setup-apollo-client`
- `git checkout setup-apollo-client`
- `cd client` - change to the client folder
- `yarn start`
- navigate to `localhost:3000`
- A page should be displayed

**Any background context you want to provide?**

- None

**Screenshots (if appropriate)**

- None

**Questions:**

- None
